scripts/scil_compute_connectivity.py

```diff
@@ -27,6 +28,13 @@
 pre-computed maps (LABEL1_LABEL2.nii.gz) following the same naming convention
 as the input directory. Each will generate a matrix. The average non-zeros
 value in the map will be reported in the matrices nodes.
+
+The parameters --lesion_load will compute 3 lesion related matrices:
+lesion_count.npy, lesion_vol.npy, lesion_sc.npy and put it inside of a
+specified folder. They represent the number of lesion, the total volume of
+lesion and the total of streamlines going through the lesion for  of each
```

Comment:
the total number of streamlines (streamline count, sc) going through 

should lesion be replaced by lesion(s) everywhere?

Reply:
It was lesions before and you said to switch it to lesion.
I will put lesion(s) everywhere in the docstring but variable name will stay `lesion`.
Is that ok?

Reply:
yes ok